### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by 8Men5Camels**

### CODE REVIEW

**Comment by 8Men5Camels** [comment]
не знаю в чем ошибка... и есть ли она

File: app/main.py [hunk 1]
```diff
@@ -15,3 +15,8 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :return: count occurrences of letter in phrase
     """
     # write your code here
```
Comment (danylott) [inline app/main.py:17]: Please, remove this comment, when you implement your solution

File: app/main.py [hunk 2]
```diff
@@ -15,3 +15,8 @@ def count_occurrences(phrase: str, letter: str) -> int:
...
+    count = 0
```
Comment (danylott) [inline app/main.py:18]: Add new line here to split `variable creation` and `loop iteration` logic

File: app/main.py [hunk 3]
```diff
@@ -15,3 +15,8 @@ def count_occurrences(phrase: str, letter: str) -> int:
...
+    for i in phrase:
+        if i.lower() == letter.lower():
+            count += 1
```
Comment (danylott) [inline app/main.py:21]: here also add new line please

**Review by danylott — CHANGES_REQUESTED** [review]

**Review by danylott — APPROVED** [review]

**Pull request closed without merging**